02/d3prof3t/game.py

```diff
@@ -0,0 +1,84 @@
+#!python3
+# Code Challenge 02 - Word Values Part II - a simple game
+# http://pybit.es/codechallenge02.html
+
+import itertools
+import random
+
+from data import DICTIONARY, LETTER_SCORES, POUCH
+NUM_LETTERS = 7
+
+def draw_letters():
+    """Pick NUM_LETTERS letters randomly. Hint: use stdlib random"""
+    return random.sample(POUCH, NUM_LETTERS)
+
+def input_word(draw):
+    """Ask player for a word and validate against draw.
+    Use _validation(word, draw) helper."""
+    word = input()
+    if word:
+        if _validation(word, draw):
```

Comment:
Maybe better: try _validation exception ValueErrror return None?

Reply:
Thanks for the review!
This seems to be a good pythonic way to do it. Will update and send the PR right away

Reply:
PR >> https://github.com/pybites/challenges/pull/234